docs/azure_arc_jumpstart/azure_arc_k8s/developer/dapr/dapr_statestore/_index.md

```diff
@@ -0,0 +1,205 @@
+
+    > **NOTE: The Jumpstart scenarios are designed with as much ease of use in-mind and adhering to security-related best practices whenever possible. It is optional but highly recommended to scope the service principal to a specific [Azure subscription and resource group](https://docs.microsoft.com/cli/azure/ad/sp?view=azure-cli-latest) as well considering using a [less privileged service principal account](https://docs.microsoft.com/azure/role-based-access-control/best-practices)**
+
+## Automation Flow
+
+For you to get familiar with the automation and deployment flow, below is an explanation.
+
+- User has deployed the Kubernetes cluster and connected it as an Azure Arc-enabled Kubernetes cluster.
+
+- User is editing the environment variables in the below Shell script file (1-time edit) which then be used throughout the configuration.
+
+- User is running the shell script. The script will use the extension management feature of Azure Arc to deploy the Dapr extension.
+
+- The script will also create and configure the state store (Azure Cache for Redis), and deploy the Node.js app with the sidecar.
+
+- User is verifying the cluster and ensuring the Dapr extension is deployed.
+
+- User is verifying the service and confirming the order was persisted in the state store.
+
+- Additionally, the User will deploy the Python app with the Dapr sidecar and observe messages to confirm successful persistence.
+
+## Deployment
+
+To create a new extension Instance, we will use the _k8s-extension create_ command while passing in values for the mandatory parameters. This scenario provides you with the automation to deploy the Dapr extension on your Azure Arc-enabled Kubernetes cluster.
+
+> **NOTE: Before installing the Dapr extension, make sure that the _kubectl_ context is pointing to your Azure Arc-enabled Kubernetes cluster. To do that, you can refer to the [official Kubernetes documentation](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) to find the options to change the kubecontext to different Kubernetes clusters.**
+
+![Screenshot showing current kubectl context pointing to K8s cluster](./03.png)
+
+- In the screenshot below, notice how currently there are no extensions installed yet in your Arc-enabled Kubernetes cluster.
+
+    ![Screenshot showing Azure Portal with Azure Arc-enabled Kubernetes resource extensions](./04.png)
+
+- Edit the environment variables [in the script](https://github.com/microsoft/azure_arc/blob/main/azure_arc_k8s_jumpstart/multi_distributions/dapr/dapr_k8s_statestore.sh) to match your environment parameters followed by running the _`. ./dapr_k8s_statestore.sh`_ command.
+
+    ![Screenshot parameter](./05.png)
+
+    ![Screenshot parameter examples](./06.png)
```

Comment:
Check the previous comment on adding comments :-) and update screenshots

Reply:
Updated